contracts/upgradeable_contracts/RewardableValidators.sol

```diff
@@ -54,11 +54,11 @@ contract RewardableValidators is BaseBridgeValidators {
         return isInitialized();
     }
 
-    function addValidator(address _validator, address _reward) external onlyOwner {
+    function addRewardableValidator(address _validator, address _reward) external onlyOwner {
```

Comment:
We need to redefine `addValidator` in this contract as so it cause `revert()` if it is called. Such redefinition will keep data consistency if it is called occasionally.

Reply:
I'm not sure I understood this correctly. `addValidator(address _validator)` method is defined on `BridgeValidators` contract, not `BaseBridgeValidators`.  So the only method to add a validator on `RewardableValidators` contract is `addRewardableValidator(address _validator, address _reward)`

Reply:
Yes, you are correct! sorry for misunderstanding.